operations/services.md

```diff
@@ -184,6 +184,107 @@ A positive value for `EndOffset` is normal for a nightly stay and implies that t
+
+### Response
+
+```javascript
+{
+  "TimeUnitStartsUtc": [
+    "2024-01-31T23:00:00Z",
+    "2024-02-01T23:00:00Z",
+    "2024-02-02T23:00:00Z",
+    "2024-02-03T23:00:00Z",
+    "2024-02-04T23:00:00Z"
+  ],
+  "ResourceCategoryAvailabilities": [
+    {
+      "ResourceCategoryId": "d1801d11-fe8d-404b-a26f-af170189605a",
+      "Metrics": {
+        "OutOfOrderBlocks": [0, 1, 0, 0, 1],
+        "PublicAvailabilityAdjustment": [7, 5, 4, 3, 4],
+        "OtherServiceReservationCount": [0, 0, 1, 0, 0],
+        "Occupied": [7, 5, 4, 3, 4],
+        "ConfirmedReservations": [7, 5, 4, 3, 4],
+        "OptionalReservations": [0, 2, 0, 0, 1],
+        "BlockAvailability": [0, 0, 1, 0, 0],
+        "AllocatedBlockAvailability": [0, 0, 0, 1, 0],
+        "UsableResources": [8, 8, 8, 8, 8],
+        "ActiveResources": [8, 8, 8, 8, 8]
+      }
+    }
+  ]
+}
+```
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `TimeUnitStartsUtc` | array of string | required | Set of all time units covered by the time interval; expressed in UTC timezone ISO 8601 format. |
+| `ResourceCategoryAvailabilities` | array of [Resource category availability](#resource-category-availability-ver-2024-01-22) | required | Resource category availabilities. |
+
+#### Service availability metrics
+
+* `OutOfOrderBlocks` - Number of resources that are out of order for the resource category ([resource block](./resourceblocks.md)).
+* `PublicAvailabilityAdjustment` - Number of resources marked as manual [availability adjustments](./availabilityadjustments.md).
+* `OtherServiceReservationCount` - Number of resources occupied by another service.
+* `Occupied` - Number of bookings that have been assigned to the resource category (i.e. reservations and blocks).
+* `ConfirmedReservations` - Number of confirmed reservations that have been assigned to the resource category.
+* `OptionalReservations` - Number of optional reservations that have been assigned to the resource category.
+* `BlockAvailability` - Number of blocked resources (from an availability block / allotment).
+* `AllocatedBlockAvailability` - Number of blocked resources that are in a deducting state (from an availability block / allotment).
+* `ActiveResources` - Number of active resources.
+* `UsableResources` - Number of usable resources (i.e. which are not out of order).
+* …
+
+#### Resource category availability (ver 2024-01-22)
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `CategoryId` | string | required | Unique identifier of the [Resource category](resources.md#resource-category). |
+| `Metrics` | dictionary | required | Dictionary with [Service availability metrics](#service-availability-metrics). Key corresponds to the metric name, value is an array of numbers with metric values for covered dates. |
```

Comment:
Correct me if I'm wrong, but I think Type `dictionary` should be ```[Dictionary](_objects.md#dictionary)```, as it's not an official JSON type but one that we define ourselves.

Reply:
Oops, realised my mistake – `Dictionary` as defined by us is strings not numbers, so a new object type is needed.